frontend/src/components/common/podcast-filter/helpers/get-current-values.helper/get-current-values.helper.ts

```diff
@@ -0,0 +1,11 @@
+import { Genre, PodcastLoadFilter } from 'common/types/types';
```

Comment:
move it to `frontend/src/components/common/podcast-filter/helpers/get-current-values/get-current-values.helper.ts`